sgv2-filter.user.js

```diff
@@ -470,6 +508,78 @@ function createFilterUiFilterOptionsRow() {
   flexGrowLeftDiv.appendChild(minLevelToDisplayInput);
   flexGrowLeftDiv.appendChild(levelDashSpan);
   flexGrowLeftDiv.appendChild(maxLevelToDisplayInput);
+    
+  // The "minimal points to display" number input
+  var minPointsToDisplayInput = document.createElement("input");
+  minPointsToDisplayInput.setAttribute("type", "number");
+  minPointsToDisplayInput.setAttribute("maxLength", "3");
+  minPointsToDisplayInput.style.width = "62px";
+  minPointsToDisplayInput.value = minPointsToDisplay;
+  minPointsToDisplayInput.onchange = function() {
+    // Filter out invalid values
+    var minPointsToDisplayInputValue = parseInt(minPointsToDisplayInput.value);
+    if (minPointsToDisplayInputValue < 0 || minPointsToDisplayInputValue > 150 || minPointsToDisplayInputValue > maxPointsToDisplay) {
+      minPointsToDisplayInput.value = minPointsToDisplay;
+    } else if (minPointsToDisplay != minPointsToDisplayInputValue) {
+      // If the value changed, save it and update the UI
+      GM_setValue(KEY_MIN_POINTS_TO_DISPLAY, minPointsToDisplayInputValue);
+      minPointsToDisplay = minPointsToDisplayInputValue;
+      updateFilterCaption();
+      filterGiveaways();
+    }
+  };
+  // Accept only digits
+  minPointsToDisplayInput.onkeypress = function(event) {
+    return isDigit(event.charCode);
+  };
+
+  // The "maximal level to display" number input
+  var maxPointsToDisplayInput = document.createElement("input");
+  maxPointsToDisplayInput.setAttribute("type", "number");
+  maxPointsToDisplayInput.setAttribute("maxLength", "3");
+  maxPointsToDisplayInput.style.width = "62px";
+  maxPointsToDisplayInput.value = maxPointsToDisplay;
+  maxPointsToDisplayInput.onchange = function() {
+    // Filter out invalid values
+    var maxPointsToDisplayInputValue = parseInt(maxPointsToDisplayInput.value);
+    if (maxPointsToDisplayInputValue < 0 || maxPointsToDisplayInputValue < minPointsToDisplay) {
+      maxPointsToDisplayInput.value = maxPointsToDisplay;
+    } else if (maxPointsToDisplayInputValue > 150) {
```

Comment:
Use the DEFAULT_MAX_POINTS_TO_DISPLAY constant instead of a hardcoded 150 value. When this will change, we'll just change the constant value and it'll work again.